src/index.html

```diff
@@ -8,6 +8,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
```

Comment:
Each star in a block should have a class `stars__star` and no extra classes or inline styles. Fix classname